tests/drivers/can/api/src/classic.c

```diff
@@ -486,6 +486,31 @@ ZTEST_USER(can_classic, test_bitrate_limits)
 	zassert_true(min <= max, "min bitrate must be lower or equal to max bitrate");
 }
 
+/**
+ * @brief Test setting a too low bitrate.
+ */
+ZTEST_USER(can_classic, test_set_bitrate_too_low)
```

Comment:
There is already negative test that checks error code returned by can_set_bitrate() when bitrate value is too high:
https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/drivers/can/api/src/classic.c#L492
Here, I add similar test. It checks that can_set_bitrate() returns -ENOTSUP when bitrate value is too low.